### ISSUE TITLE
Cannot leave out port number anymore (regression) in 2.7

### ISSUE THREAD

**Issue by bolkedebruin** [psycopg/psycopg2#523]
Port number now seem to be required when creating connections. This is a regression (imho) towards 2.6
 
```File "/home/travis/build/bolkedebruin/airflow/.tox/py27-cdh-airflow_backend_sqlite/lib/python2.7/site-packages/psycopg2/__init__.py", line 130, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
OperationalError: invalid port number: "None"```

**Comment by dvarrazzo** [comment]
Seems like a duplicate of #517.

The problem is already fixed, will be released in 2.7.1.

**Comment by dvarrazzo** [comment]
Sorry, didn't explain how to solve the problem. Problem is that None arguments are taken as a `None` string instead of being ignored as they were before. So just leave `port` or any other argument of value `None` out of `connect()`.
